api/app/controllers/mno_enterprise/jpi/v2/users_controller.rb

```diff
@@ -1,8 +1,24 @@
 module MnoEnterprise
   class Jpi::V2::UsersController < Jpi::V2::ApiController
+    def show
+      resp = MnoHubClient.get(File.join(endpoint, params.require(:id)), filter_params(params).merge(authentication_hash))
+      render body: apply_intercom_auth(resp), content_type: 'application/vnd.api+json', status: response.code
+    end
+
     def update_password
       resp = MnoHubClient.patch(File.join(endpoint, params.require(:id), 'update_password'), {body: request.raw_post}.merge(authentication_hash))
       render_results(resp)
     end
+
+    private
+
+    def apply_intercom_auth(response)
+      user_hash = current_user.intercom_user_hash
```

Comment:
When Intercom is disable you'll have `no method error intercom_user_hash`?

Reply:
@ouranos in 5.0 the `IntercomUser` concern is always included https://github.com/maestrano/mno-enterprise/blob/5.0/core/app/models/mno_enterprise/user.rb#L3. 

Please advise? This change will be merged into 5.0 right? Should I add `MnoEnterprise.intercom_enabled?` guarding to the v2/users_controller?

Reply:
Ok my bad.

Yes better to had the guard then